core/tests/ts-integration/tests/custom-account.test.ts

```diff
@@ -95,6 +113,94 @@ describe('Tests for the custom account behavior', () => {
         ).toBeAccepted([erc20BalanceChange, feeCheck]);
     });
 
+    test('Should fail transaction validation due to timestamp assertion in the validation tracer - short time window', async () => {
+        const now = Math.floor(Date.now() / 1000);
+        const rangeStart = now - 60;
+        const rangeEnd = now + 60;
```

Comment:
How do we know that this range is too short? Maybe, it would make sense to read node config and then create a shorter range? Ditto for the next test.